dummy/controllers/mealController/mealController.js

```diff
@@ -52,4 +52,37 @@ export default class Meal {
   static getAllMeal(req, res) {
     getAllMeals(res, mealDb);
   }
+
+  // Update meal option
+
+  /**
+   *
+   * @param {*} req  THE REQUEST OBJECT
+   * @param {*} res   THE RESPONSE OBJECT
+   * @param {*} json  THE  RETURN JSON
+   */
+  static updateMeal(req, res) {
+    let updatedMeal;
+    const { id } = req.params;
+    mealDb.forEach((meals) => {
+      if (meals.id === parseInt(id, 10)) {
```

Comment:
This approach assumes that every time an update is to be made, the user wants to update all fields, which would not always be the case. You also want to handle cases where the user would update just a couple of information on the meal object.
```javascript
meals.img = req.body.img || meals.img
meals.desc = req.body.description || meals.desc;
meals.title = req.body.title || meals.title;
meals.price = req.body.price || meals.price;
```